### ISSUE TITLE
More precise tso time decoding.

### ISSUE THREAD

**Issue by winkyao** [pingcap/pd#1241]
https://github.com/pingcap/pd/blob/master/pdctl/command/tso_command.go#L51

```
physicalTime := time.Unix(int64(physical/1000), 0)
```

Should change to 

```
physicalTime := time.Unix(int64(physical/1000), physical%1000 * 1e6)
```

**Comment by disksing** [comment]
PTAL @nolouch

**Comment by rleungx** [comment]
Closed via #1242.
